.github/ISSUE_TEMPLATE/copy.yml

```diff
@@ -0,0 +1,41 @@
+name: "Copy"
+description: Additions or updates to the copy
+title: "🔤 "
+projects: ["GCTC-NTGC/8"]
+type: "Debt"
+labels: copy
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this copy issue!
+  - type: textarea
+    id: description
+    attributes:
+      label: 🔤 Copy
+      description: Describe the update or addition to the copy that you are proposing.
+    validations:
+      required: true
+  - type: textarea
+    id: details
+    attributes:
+      label: 🕵️ Details
+      description: Add any additional details that could assist with the implementation of the new or existing copy.
+  - type: textarea
```

Comment:
Does the copy template need a proposed solution section?  Seems unnecessary to me.